packages/tools/pie-css/package.json

```diff
@@ -7,11 +7,15 @@
     "index.css",
     "scss"
   ],
-  "type": "module",
-  "main": "index.css",
+  "main": "dist/index.css",
   "scripts": {
-    "lint:style": "run -T stylelint ./**/*.{css,scss} --allow-empty-input",
-    "lint:style:fix": "yarn lint:style --fix"
+    "build:css": "run -T ts-node ./build-css.ts",
```

Comment:
do you mind sharing why we didn't use vite for this package and decided to use ts-node instead?

Reply:
we aren't packaging any javascript for consumers @raoufswe

Reply:
I believe `vite` isn't limited to packaging for consumers only. it can be used to transpile your typescript code to javascript code and I think this is what we are using ts-node for here.

the flow would be, similar to the other packages, have `build-css` as a typescript file, when you build / or run, run vite to transpile your code to javascript and run the transpiled code from the dist folder. you can find more details in the generator setup, but my concern is we shouldn't add two packages to the repo that do the same thing if possible.

Reply:
I think there's a bit of a (very understandable) misunderstanding here. The only dist output is an `index.css` file :) We aren't compiling the TS. It's just running in Node to build the CSS file. Therefore we don't need a vite config